crates/protocol/src/utils.rs

```diff
@@ -0,0 +1,29 @@
+//! Utility methods used by protocol types.
+
+/// Returns if the given `value` is a deposit transaction.
+pub fn is_deposit<B>(value: &B) -> bool
```

Comment:
perhaps rename this to `starts_with_2781_deposit` or smth?

Reply:
Ah nice, will do